build.gradle

```diff
@@ -217,12 +219,11 @@ dependencies {
     testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-json'
 
     testImplementation group: 'org.springframework.cloud', name: 'spring-cloud-contract-wiremock', {
-        exclude group: 'com.github.tomakehurst', module: 'wiremock-jre8-standalone'
+        exclude group: 'com.github.tomakehurst', module: 'wiremock-standalone'
     }
-    testImplementation group: 'com.github.tomakehurst', name:'wiremock-jre8', version: '2.26.3'
```

Comment:
I haven't looked closely at this but wouldn't this just be a module that gives new features that requires jdk8?